test/testdrive/mz-depends.td

```diff
@@ -0,0 +1,63 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Test `mz_internal.mz_dependencies`.
+
+> CREATE SOURCE with_subsources FROM LOAD GENERATOR AUCTION FOR ALL TABLES;
+
+> SELECT
```

Comment:
It'd be cool for this to be a `SHOW DEPENDENCIES` command or something in the future.